core/dbt/cli/flags.py

```diff
@@ -172,6 +177,12 @@ def assign_params(ctx, params_assigned_from_default):
         for param in params:
             object.__setattr__(self, param.lower(), getattr(self, param))
 
+    # If the value of the lead parameter was set explicitly, apply the value to follow,
+    # unless follow was also set explicitly.
```

Comment:
tiny nit: documenting the method in a docstring below the signature would be more pythonic + consistent with the rest of this file.